src/components/CustomTileModal/CustomTileModal.tsx

```diff
@@ -45,11 +45,21 @@ const CustomTileModal: React.FC<CustomTileModalProps> = ({
     const [displayHeader, setDisplayHeader] = useState(
         selectedItem?.displayHeader ?? '',
     )
+    const [errorMessage, setErrorMessage] = useState<string | null>(null)
 
     const [isSubmitAttempted, setIsSubmitAttempted] = useState(false)
 
     const handleSubmit = (actionType: ActionType) => {
-        setIsSubmitAttempted(true)
+        if (!sourceUrl.endsWith('.no')) {
```

Comment:
Skal man bare få lov til å lenke til '.no' sider?

Reply:
Godt spørsmål, men ragnhild skrev dette: 

Legge til validering av feltet og gi følgende feilmelding om man skriver inn inn noe som ikke er gyldig: Lenkeadressen er ugyldig. Skriv inn en nettadresse på formatet [nettside.no](http://nettside.no/) eller [www.nettside.no](http://www.nettside.no/).

Jeg antar at det ikke vil være relevant å ha noen .com-tavla sider?

Reply:
hmm.. for meg virker det som eksempler, og ikke at man bare skal kunne lenke til .no-sider. Du kan jo høre med henne ☺️

Reply:
Slenger på en .com også jeg 😎

Reply:
👍